components/prism-rescript.js

```diff
@@ -0,0 +1,60 @@
+Prism.languages.rescript = {
+	'comment': {
+		pattern: /\/\/.*|\/\*[\s\S]*?(?:\*\/|$)/,
+		greedy: true
+	},
+	'char': { pattern: /'(?:[^\r\n\\]|\\(?:.|\w+)')/, greedy: true },
```

Comment:
Oh no, sorry. I mistyped.
```suggestion
	'char': { pattern: /'(?:[^\r\n\\]|\\(?:.|\w+))'/, greedy: true },
```

Reply:
I'll fix

Reply:
Done